src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -843,7 +847,7 @@ def create_nic(resource_group_name, network_interface_name, subnet, location=Non
                load_balancer_backend_address_pool_ids=None,
                load_balancer_inbound_nat_rule_ids=None,
                load_balancer_name=None, network_security_group=None,
-               private_ip_address=None, private_ip_address_version=IPVersion.ipv4.value,
+               private_ip_address=None, private_ip_address_version=None,
```

Comment:
By doing this, and not updating the params file, now the default will not be show on supported API versions!